src/applications/mhv/medical-records/components/shared/DownloadingRecordsInfo.jsx

```diff
@@ -21,7 +20,7 @@ const DownloadingRecordsInfo = props => {
     >
       {/* Use the "!== false" syntax because checking the inverse causes the component to be resized incorrectly
             when the value changes from undefined to true. */}
-      {allowTxtDownloads !== false ? (
+      {allowTxtDownloads ? (
```

Comment:
Why the change to this logic? Did you test this out with the condition listed on the previous lines? If you did and it works correctly, then please remove the comment.